ADD-ONS-CHANGELOG.md

```diff
@@ -0,0 +1,59 @@
+# React Add-Ons Change Log
+
+As of React v15.5.0 we discontinued active maintenance of the React Addons
+packages. In previous versions of React we kept the version numbers of
+these packages in sync with the React version, and included updates to
+them in the main React `CHANGELOG.md`.
+
+Since the release versions of these packages are now out of sync with React,
+and we are no longer including them as a supported part of React, we have split
+out the latest changes into this separate change log.
+
+## Create Fragment
+
+### 15.5.3
+
```

Comment:
In general we avoid deeply technical descriptions of changes.
I would prefer that instead of "envify and collapse" we would say something like

* Fixed the accidental size regression in UMD bundles.